script.js

```diff
@@ -1,11 +1,22 @@
-// Create an array called `colors` that contains three colors.
+var colors = ['green', 'yellow', 'red'];
 
 // Log the last item in the array to the console.
 
-// Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
+   console.log(colors[2]);
 
+// Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
+for(i = 0; i < colors.length; i++){
+  var text = "";
+  text += "My favoriate color is" + " " + colors[i];
+}
 // Create an `instructor` object that contains three key-value pairs.
+var instructor = {
+name: "Andy",
+program: "WDI",
+gender:"male"
+};
 
 // Add a `facial-hair` property to `instructor`.
-
+instructor.appearance = "facial-hair"
```

Comment:
Here you're adding a value, the string `"facial-hair"` to the property `appearance`. We were looking for `"facial-hair"` to be the property name like this

```js
instructor["facial-hair"] = "scruffy";
```